<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/res/cat3.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="level1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/cat.jpg);</string>
    </property>
    <property name="text">
     <string>关卡1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/cat1.jpg);</string>
    </property>
    <property name="text">
     <string>关卡2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/cat2.jpg);</string>
    </property>
    <property name="text">
     <string>关卡3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/cat3.jpg);</string>
    </property>
    <property name="text">
     <string>关卡4</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
